x-pack/plugins/actions/server/feature.ts

```diff
@@ -71,4 +69,34 @@ export const ACTIONS_FEATURE = {
       ui: ['show', 'execute'],
     },
   },
+  subFeatures: [
+    {
+      name: i18n.translate('xpack.actions.featureRegistry.edrSubFeatureName', {
+        defaultMessage: 'Endpoint Security',
+      }),
+      description: i18n.translate('xpack.actions.featureRegistry.edrSubFeatureDescription', {
+        defaultMessage: 'Includes: Sentinel One, Crowdstrike',
+      }),
+      privilegeGroups: [
+        {
+          groupType: 'independent',
+          privileges: [
+            {
+              api: [EDR_EXECUTE_PRIVILEGE_API_TAG],
+              id: 'edr_execute',
+              name: i18n.translate('xpack.actions.featureRegistry.edrSubFeaturePrivilege', {
+                defaultMessage: 'Execute',
+              }),
+              includeIn: 'all',
+              savedObject: {
+                all: [ACTION_TASK_PARAMS_SAVED_OBJECT_TYPE, CONNECTOR_TOKEN_SAVED_OBJECT_TYPE],
+                read: [ACTION_SAVED_OBJECT_TYPE],
+              },
```

Comment:
__question:__ I admit I don’t know much about Alerting RBAC, but why do we need to add these SO types here? I see that both `read` and `all` base privileges already grant the same access, and sub-feature privileges are always granted together with one of the base privileges.

Reply:
Resolved in this commit [f62d3e8](https://github.com/elastic/kibana/pull/203503/commits/f62d3e8f43b6939f1cefe20a7bd4a8c22a44cb61)